src/ensembl/src/shared/accordion/components/AccordionItem.test.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { cleanup, render } from 'react-testing-library';
```

Comment:
But we aren't using `react-testing-library`.
Either enzyme for all unit tests, or react-testing-library for all unit tests, but not both.

Reply:
Yes, I'm currently re-writing all the tests.